src/app.component.js

```diff
@@ -5,17 +5,18 @@ import { Main } from './component/main';
 import { Footer } from './component/footer';
```

Comment:
Why are you naming this component from the lower case? Will be better if you're going to use capital letters when name the components, instead of mixing two styles

Reply:
And about app.component.js, please rename it to app.js instead, because it's a component anyway.

Reply:
component called 'Footer', the folder where placed component calls footer. I think it's normal situation when the folder named from lowercase